### PR TITLE
Feature/cele 27 Layout manager wrapper component

### PR DESCRIPTION
**Pull request by vidhyalongani**
Screen Shot 2024-05-28 at 3 51 51 PM
Screen Shot 2024-05-28 at 3 51 35 PM
Screen Shot 2024-05-28 at 3 51 30 PM

### CODE REVIEW

File: applications/visualizer/frontend/src/components/ViewerContainer/CustomSwitch.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { useState } from "react";
 
 const { white, brand600, gray100 } = vars;
 
-const CustomSwitch = () => {
+const CustomSwitch = ({width, height, thumbDimension, checkedPosition}) => {
```
Comment (aranega) [inline applications/visualizer/frontend/src/components/ViewerContainer/CustomSwitch.tsx]: As the signature of the component changed, the new parameters should be also added when it is used in this component ([here](https://github.com/MetaCell/c-elegans-app/blob/05b6556bf95e398b15f8b12b5b0a0cbcac463ac5/applications/visualizer/frontend/src/components/ViewerContainer/CustomListItem.tsx#L54)).

I can see that the parameter are kind of optional, so to not modify the `CustomListItem` component to add the parameters, another solution would be to define default values, or an interface stating that the fields are optional :)

**Review by aranega — CHANGES_REQUESTED** [review]
Thanks for the PR @vidhya-metacell ! All looks good to me. There is only a small issue while compiling the project, almost nothing, either some default value missing, or the parameters of a component to declare as optional and it should be perfect like this :)

File: applications/visualizer/frontend/src/components/ViewerContainer/CustomSwitch.tsx (hunk 1 above)
Comment (vidhyalongani) [reply applications/visualizer/frontend/src/components/ViewerContainer/CustomSwitch.tsx]: Hey @aranega , thanks for the suggestion. It makes sense.  I have made the changes.

**Review by aranega — APPROVED** [review]
Thanks for the fix!

**Pull request merged**